src/components/shared/ReactFlow/FlowCanvas/TaskNode/TaskNodeCard/TaskNodeCard.tsx

```diff
@@ -170,9 +170,18 @@ const TaskNodeCard = () => {
       ref={nodeRef}
     >
       <CardHeader className="border-b border-slate-200 px-2 py-2.5 flex flex-row justify-between items-start">
-        <CardTitle className="break-words text-left text-xs text-slate-900">
-          {name}
-        </CardTitle>
+        <div className="flex flex-col">
```

Comment:
You can use our UI Primitives!


```suggestion
        <BlockStack>
```